src/main.ts

```diff
@@ -31,10 +85,9 @@ async function bootstrap() {
   app.enableCors();
   app.setGlobalPrefix('api/v1', { exclude: ['/', 'health', 'api', 'api/v1', 'api/docs'] });
 
-  // TODO: set options for swagger docs
   const options = new DocumentBuilder()
-    .setTitle('<project-title-here>')
-    .setDescription('<project-description-here>')
+    .setTitle('Swagger')
```

Comment:
This is not meant to be. The project Title has been set in a different PR

Reply:
okay, I will do that now